### PR TITLE
ARTESCA-2606 - fix ui crashed when alert manager is not available

### PR DESCRIPTION
**Pull request by MonPote**

### CODE REVIEW

**Review by JBWatenbergScality — APPROVED** [review]

File: ui/src/hooks.js [hunk 1]
```diff
@@ -101,7 +101,7 @@ export const useVolumesWithAlerts = (nodeName?: string) => {
   //This forces alerts to have been fetched at least once (watchdog alert should be present)
   // before rendering volume list
   // TODO enhance this using useAlerts status
-  if (alerts.length === 0) return [];
+  if (!alerts || alerts.length === 0) return [];
```
Comment (gdemonet) [inline ui/src/hooks.js:104]: Can't we use null coalescing for this?

```suggestion
  if (alerts?.length ?? 0 === 0) return [];
```

Or just rely on the "falsiness" of `0`, as is done in the other change below:

```suggestion
  if (!alerts?.length) return [];
```

File: ui/src/services/alertUtils.js [hunk 2]
```diff
@@ -175,7 +175,7 @@ export const getHealthStatus = (
   alerts: Alert[],
   activeOn?: string = new Date().toISOString(),
 ): Health => {
-  if (!alerts.length) return STATUS_HEALTH;
+  if (!alerts || !alerts.length) return STATUS_HEALTH;
```
Comment (gdemonet) [inline ui/src/services/alertUtils.js:178]:
```suggestion
  if (!alerts?.length) return STATUS_HEALTH;
```

File: ui/src/hooks.js (hunk 1 above)
Comment (MonPote) [reply ui/src/hooks.js:104]: We can but I have the feeling that it's a bit difficult to read.
Especially in this case : `alerts?.length ?? 0 === 0`.
```diff
-  if (!alerts || alerts.length === 0) return [];
+  if (alerts?.length ?? 0 === 0) return [];
```

I choose to just add `!alerts ||` for consistency but if you want I can change at least : 
```diff
- if (!alerts || !alerts.length) return STATUS_HEALTH;
+ if (!alerts?.length) return STATUS_HEALTH;
```

**Pull request merged**